app/components/Button/index.tsx

```diff
@@ -48,7 +48,7 @@ const Button = ({
   const LinkComponent = !action.startsWith('http')
     ? Link
     : ({to, children, ...props}: {[k: string]: any}) => (
-        <a href={to} {...props}>
+        <a href={to} rel="noreferrer" target="_blank" {...props}>
```

Comment:
@bryceerobertson I switched it here so that by default links to other pages open in a new tab. @mruwnik is that ok?

Reply:
not according to me. That's what the control button is for... But there was a whole discussion about this and I think the external links faction won :/ @melissasamworth is that correct?